src/translations/nbTranslations.json

```
@@ -385,6 +385,7 @@
       "reserve_doi": "Reservasjon av DOI feilet",
       "search": "Søk feilet",
       "send_message": "Kunne ikke sende melding",
+      "storing_orcid_credentials": "Kunne ikke lagre orcid påloggings signatur",
```

Kanskje er dette greit nok? Brukeren har forsåvidt ikke forhold til det som skjer her uansett
```suggestion
      "storing_orcid_credentials": "Kunne ikke lagre ORCID-data",
```

FYI: Vi bruker bare å angi oversettelser til bokmål i koden nå, så har vi en annen oversetter som legger inn for nynorsk og engelsk selv i lokalise.com: https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/documentation/translations/translations.md